### PR TITLE
feat: Update window guide

### PR DESCRIPTION
**Pull request by 1ambda**
Update windows zeppelin integration guide.

### CODE REVIEW

**Comment by 1ambda** [comment]
Please help review @rbpark @AhyoungRyu

File: docs/guide/zeppelin_integration.md [hunk 1]
```diff
@@ -38,7 +40,24 @@ Don't forget to copy your **Token** generated before.
...
+
+When you connect a space to your ZEPL account, you need to set the following **environment variables** into your `ZEPPELIN_HOME/conf/zeppelin-env.cmd` file.
+If you don't have this file, you can create it from `ZEPPELIN_HOME/conf/zeppelin-env.cmd.template`.
+
+```
+$ cd ZEPPELIN_HOME/conf
+$ cp zeppelin-env.cmd.template zeppelin-env.cmd
+```
+
+After activating this configuration file,  add below three lines to `zeppelin-env.sh` file.
```
Comment (AhyoungRyu) [inline docs/guide/zeppelin_integration.md]: you mean `zeppelin-env.cmd`?

File: docs/guide/zeppelin_integration.md [hunk 2]
```diff
@@ -55,24 +74,46 @@ export ZEPPELINHUB_USER_KEY="YOUR_USER_KEY"
...
+```sh
+set ZEPPELIN_NOTEBOOK_STORAGE=org.apache.zeppelin.notebook.repo.GitNotebookRepo, org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo
+set ZEPPELINHUB_API_ADDRESS=https://www.zepl.com
+set ZEPPELINHUB_API_TOKEN=YOUR_TOKEN_STRING
+```
+
 <br/>
-#### Zeppelin-0.7.x multiple users on Zeppelin server
+
+#### Zeppelin-0.7.x multiple users on Zeppelin server for Linux, OSX
```
Comment (AhyoungRyu) [inline docs/guide/zeppelin_integration.md:116]: So Windows user can't use this feature? Just wondered ;)

Comment (1ambda) [reply docs/guide/zeppelin_integration.md:116]: I haven't tested so intentionally skip the section. Would it better to add something there? or..

File: docs/guide/zeppelin_integration.md (hunk 1 above)
Comment (1ambda) [reply docs/guide/zeppelin_integration.md]: Fixed. Thanks.

File: docs/guide/zeppelin_integration.md (hunk 2 above)
Comment (AhyoungRyu) [reply docs/guide/zeppelin_integration.md:116]: ahh the user needs to do sth with Shiro setting file in this case.. Then if we can just check whether Zeppelin <-> Shiro works well or not in Windows, it'll be fine to add sth like 

```sh

#### Zeppelin-0.7.x multiple users on Zeppelin server for Windows

set ZEPPELIN_NOTEBOOK_STORAGE=org.apache.zeppelin.notebook.repo.GitNotebookRepo, org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo
set ZEPPELINHUB_API_ADDRESS=ADDRESS_OF_ZEPPELINHUB_SERVICE (e.g. https://www.zepl.com)
```

But it's okay we can do it this in later time

Comment (1ambda) [reply docs/guide/zeppelin_integration.md:116]: Then let's do later. I guess windows is usually not used as the server. (for multi user purpose)

Comment (1ambda) [reply docs/guide/zeppelin_integration.md:116]: @AhyoungRyu Can I get the approval? If all things are ok.

**Review by AhyoungRyu — APPROVED** [review]
LGTM

**Pull request merged**